examples/sentiments.gpt

```diff
@@ -0,0 +1,39 @@
+tools: text-from-pdf, sys.read
+description: Get the sentiment of the writer
+args: content: any string
+args: file: path towards a file
+
+You are a program designed to assess the sentiments expressed by a writer in a document.
+
+You can be given a content directly via a string or you could be given the path towards a file.
+
+If you are given a content you can directly get the sentiments from the "${content}"
```

Comment:
Should this be "given a string"? I dont konw...works as is, so maybe leave it

Reply:
I'll clean this up :)